concrete/src/Application/Service/Dashboard.php

```diff
@@ -247,6 +247,8 @@ public function getDashboardBackgroundImage()
      */
     public function getIntelligentSearchMenu()
     {
+        $loc = Localization::getInstance();
+        $loc->setActiveContext('system');
```

Comment:
This should be using different context `ui`. Also, we should use the newly available constants for the contexts, so this would be `\Concrete\Core\Localization\Localization::CONTEXT_UI` (but please import the class instead of defining it inline.